include/powsybl/iidm/converter/Parameter.hpp

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#ifndef POWSYBL_IIDM_CONVERTER_PARAMETER_HPP
+#define POWSYBL_IIDM_CONVERTER_PARAMETER_HPP
+
+#include <initializer_list>
+#include <set>
+#include <string>
+#include <vector>
+
+#include <boost/any.hpp>
+
+namespace powsybl {
+
+namespace iidm {
+
+namespace converter {
+
+class Parameter {
+public:
+    enum class ParameterType : char {
```

Comment:
If you declare this enum inner the Parameter class, rename it to Type (Parameter::Type), or extract it to its own header